.gitignore

```diff
@@ -12,5 +12,4 @@ generated/
 \#*\#
 *.swp
 /debug/
-/antlr-3.4
-deps/
+/deps/
```

Comment:
Is this even required if we have the dependencies in build/?

Reply:
We decided to build "external tools" (like LFSC) in `/deps/`. I'd keep it.